exercises/transpose/transpose_test.py

```diff
@@ -0,0 +1,275 @@
+import unittest
+from transpose import transpose
+
+
+class TransposeTests(unittest.TestCase):
+    def test_empty_string(self):
+        input_line = ""
+        expected = ""
+        self.assertEqual(
+            expected,
```

Comment:
The parameters in all `assertEqual()` calls need to be switched. Details and the background can be found here: #440